mbtools/models.py

```diff
@@ -214,6 +214,13 @@ def location(self):
         lesson_name = self.etree.xpath("../../name")[0].text
         return f"{lesson_name} (page: {self.name})"
 
+    def page_type(self):
+        qtype = self.etree.xpath("qtype")[0].text
+        if qtype == '3':
+            return 'multichoice'
+        if qtype == '20':
+            return 'content'
```

Comment:
Can we raise an `Exception` here if we hit an unexpected value? That will force us to take a closer look if we ever get a value other than these two.